watertap/unit_models/tests/test_gac.py

```diff
@@ -242,12 +255,12 @@ def gac_frame_robust(self):
             molar_volume_data={("Liq", "TCE"): 9.81e-5},
         )
         mr.fs.properties.visc_d_phase["Liq"] = 1.3097e-3
-        mr.fs.properties.dens_mass_const = 1000
-
+        mr.fs.properties.dens_mass_const = 999.7
         mr.fs.unit = GAC(
             property_package=mr.fs.properties,
-            film_transfer_coefficient_type="calculated",
-            surface_diffusion_coefficient_type="calculated",
+            film_transfer_coefficient_type="fixed",
+            surface_diffusion_coefficient_type="fixed",
+            finite_elements_ss_approximation=9,
```

Comment:
Did you notice any instability when setting number of elements too high/too low?

Reply:
9 was selected here because it gave increments closest to 0.1 which is what I had the exact solutions to in an offline spreadsheet to validate the results,. It is stable throughout, just adds 4 variables every 1 more element. The default is set to lower because lower tends to be a good enough approximation.